### PR TITLE
Take into account citation when transform a quote to paragraph

### PR DESCRIPTION
**Pull request by oandregal**
When transforming a quote to a paragraph the citation was left out. This PR makes the citation a new paragraph.

### CODE REVIEW

File: packages/block-library/src/quote/index.js [hunk 1]
```diff
@@ -97,8 +97,8 @@ export const settings = {
 			{
 				type: 'block',
 				blocks: [ 'core/paragraph' ],
-				transform: ( { value } ) =>
-					split( create( { html: value, multilineTag: 'p' } ), '\u2028' )
+				transform: ( { value, citation } ) =>
```
Comment (iseulde) [inline packages/block-library/src/quote/index.js]: This can be added after split, no? No need to parse it.

**Comment by iseulde** [comment]
Could you added an e2e test as well?

**Comment by nosolosw** [comment]
@iseulde pushed some changes to make it more robust against void cite, quotes, or both. Also updated the existing e2e tests.

**Review by ellatrix — APPROVED** [review]
Ok, looks good.

**Comment by iseulde** [comment]
@nosolosw Oh, could we do the same for pull quote?

**Comment by nosolosw** [comment]
Cool! Going to merge.

Pullquote can only be transformed to/from a quote since [a few hours ago](https://github.com/WordPress/gutenberg/pull/10683) :tada: Depending on how clean is [the queue](https://github.com/WordPress/gutenberg/milestone/66) tomorrow morning I might be able to add the pullquote<->paragraph transform (and take the cite into account).

**Comment by paaljoachim** [comment]
https://make.wordpress.org/test/
13. Add a quote block with a citation, convert it to a paragraph block, make sure the citation is still there. (10685)

Works. Should the citation come direct below the quote in the same paragraph box? Or should it as it is right now go into another paragraph box below the quote?

**Comment by nosolosw** [comment]
> Should the citation come direct below the quote in the same paragraph box? Or should it as it is right now go into another paragraph box below the quote?

The expected behavior is that a new paragraph is created for the cite.

**Comment by paaljoachim** [comment]
Ok.

**Comment by mikelittle** [comment]
That worked for me too.

**Pull request merged**